google-feeds.html

```diff
@@ -100,7 +100,7 @@
     queryChanged: function() {
       if (this.query) {
         this.loading = true;
-        this.findFeeds();
+        this.withFeedsApi(this.findFeeds.bind(this));
```

Comment:
This is over in https://github.com/GoogleWebComponents/google-feeds/pull/4#discussion_r30004623